src/OrbitGl/TimeGraph.h

```diff
@@ -93,6 +93,8 @@ class TimeGraph : public orbit_gl::CaptureViewElement {
   void VerticallyMoveIntoView(const orbit_client_protos::TimerInfo& timer_info);
   void VerticallyMoveIntoView(Track& track);
 
+  void SetAndClampYPos(float y);
```

Comment:
I'm a bit surprised about this method being called using always a negative value of scrolling_offset? Aren't there a way to make them consistently using the same sign?

Reply:
This has now been removed